resources/views/store/partials/voucher_collectors.blade.php

```diff
@@ -38,6 +38,12 @@ class="@if($errors->has('pri_carer'))invalid @endif" type="text"
                 <i class="fa fa-plus" aria-hidden="true"></i>
             </button>
         </div>
+        <span
+            style="display:none;"
+            id="carer-name-error"
+            class="invalid-error">
+            Allowed characters: letters, spaces, hyphens and full stops.
```

Comment:
hmm. perhaps.
should we encourage them?

I'll add it.

Reply:
I'm not sure why we're letting them in the first place, but if they can, they may as well know about it in case they're trying to narrow down why the nonsense they've entered isn't working.